style/header.css

```diff
@@ -0,0 +1,97 @@
+.logo,
+.logo img {
+  margin: 3px;
+  display: inline-block;
+  width: 10vw;
+  height: auto;
+  margin-right: 10vw;
+  z-index: 1; }
+
+.sub-nav {
+  margin: 0 2% 0 auto;
+  float: right; }
+  .sub-nav .icon {
+    display: none; }
+  .sub-nav li {
+    display: inline-block;
+    margin-left: 5px; }
+    .sub-nav li .sub-nav-item {
+      font-size: 12px;
+      color: #6b6b6b;
+      text-decoration: none; }
+
+.main-nav {
+  margin: 0;
+  padding: 0;
+  list-style: none;
+  display: inline-flex;
+  width: 50vw;
+  flex-direction: row;
+  align-items: center; }
+  .main-nav .cart-icon-container {
+    background-color: rgba(238, 238, 238, 0.867);
+    text-align: left;
+    padding-left: 0; }
+    .main-nav .cart-icon-container .cart-icon {
+      display: inline-block;
+      height: 20px;
+      width: auto;
+      fill: #d10157; }
+  .main-nav a {
+    padding: 10px 15px;
+    text-align: center;
+    display: inline-block;
+    color: #6b6b6b;
+    font-size: 12px; }
+  .main-nav a:hover {
+    color: #718daa; }
+
+nav {
+  padding-top: 0.5em;
+  background-color: #fff;
+  box-shadow: 0px 0px 14px 0px rgba(0, 0, 0, 0.75); }
+
+@media (max-width: 768px) {
+  .clearfix:after {
+    content: " ";
+    display: block;
+    height: 0;
+    clear: both; }
+  .logo,
+  .logo img {
+    margin: 3px;
```

Comment:
why to repeat properties with same values for different viewport sizes?

Reply:
fixed this